### DISCUSSION TITLE
alist 能用但是用IP地址:端口号/dav， 显示Method Not Allowed，请问这个需要怎么设置。

### DISCUSSION THREAD

**Discussion by tw2266** [AlistGo/alist#2201] [Q&A]
alist 能用，但是用IP地址:端口号/dav， 显示Method Not Allowed，请问这个需要怎么设置。QNAP 的 NBS3 不能通过wedav同步到本机

**Comment by liuyuanjun** [comment]
同样的问题

**Comment by hiahia250** [comment]
同样的问题

**Comment by Skylarzz** [comment]
用浏览器访问是会看到这个的，因为 Alist webdav 没有前端
但实际上 webdav 可以正常使用

**Comment by oldsento** [comment]
openlist的情况一样，但把 http://localhost:5244/dav改成 http://localhost:5244//dav就可以了，就是加多了（/），这样浏览器打开正常，但某些支持挂载webdav的软件如aimp还是不行
